subliminal/providers/podnapisi.py

```diff
@@ -45,11 +45,12 @@ def id(self):
 
     def get_matches(self, video):
         matches = set()
-
+        # Make sure we don't compare unicode with str
+        title = self.title.encode("utf-8") if isinstance(self.title, unicode) else self.title
```

Comment:
about your comment: `I don't get why we have unicode for self.title. Should be str here.`. This is the implementation of it.